Extensions/Xtensive.Orm.Localization.Tests/ProjectionToCustomTypesTests.cs

```diff
@@ -0,0 +1,55 @@
+using System.Linq;
```

Comment:
Please add the same copyright text I posted in ```DictionaryModel``` file review.

Reply:
done